routes/media.go

```diff
@@ -296,3 +297,93 @@ func (fes *APIServer) GetFullTikTokURL(ww http.ResponseWriter, req *http.Request
 		return
 	}
 }
+
+func (fes *APIServer) UploadVideo(ww http.ResponseWriter, req *http.Request) {
```

Comment:
It's not clear to me what this endpoint does exactly.  Would you mind adding a comment above it to make it more clear?

Reply:
Added some comments - let me know if it's more clear now.

Reply:
Loooooooooooove it! Thanks :)